Fixed openInStore func support locale and added appName params

hawkup opened the pull request:
* Important breaking change
- Change parameter from appStoreId, playStoreId to Object { appName, appStoreId, appStoreLocale, playStoreId }

Commit:
Fixed openInStore func support locale and added appName params
* Important breaking change
- Change parameter from appStoreId, playStoreId to Object { appName, appStoreId, appStoreLocale, playStoreId }

FiberJW commented:
LGTM, but would you mind updating the API docs in the Readme to reflect your incoming changes? Thanks for the contributions!

Commit:
Update README
- Fix appStoreId example value on AppLink.maybeOpenURL
- Update AppLink.openInStore usage

hawkup commented:
@FiberJW Sorry for late reply. I already updated API docs.

FiberJW commented:
Thanks for the contributions! Merging now 💯
